pytest/tests/mocknet/mirror.py

```diff
@@ -237,12 +239,13 @@ def new_test_cmd(args, traffic_generator, nodes):
             f'--num-validators is {args.num_validators} but only found {len(nodes)} under test'
         )
 
-    genesis_time = new_genesis_timestamp(nodes[0])
+    ref_node = traffic_generator if traffic_generator else nodes[0]
+    genesis_time = new_genesis_timestamp(ref_node)
```

Comment:
would we ever want to run `new-test` on only a subset of nodes? I can't think of a reason really. Maybe either we should disallow it or at least print a warning/ask the user: "are you sure you want to do that?"